sys-libs/libxcrypt/libxcrypt-4.4.36-r3.ebuild

```diff
@@ -26,10 +26,6 @@ RESTRICT="!test? ( test )"
 
 DEPEND="
 	system? (
-		elibc_glibc? (
-			${CATEGORY}/glibc[-crypt(-)]
```

Comment:
`-crypt` means "crypt flag should be disabled", and `crypt(-)` means "crypt flag, but if it's not there, assume it's disabled", so `-crypt(-)` means "crypt flag should be disabled, but if it's not there, assume it's disabled".

It should be okay to keep it as-is for a bit longer to help people upgrading systems with older glibc, unless there's a problem caused by it for you?

Reply:
Your output has `-crypt(+)` which has different semantics! See f777d389830da3a28f911e38ebae51b5de299e8b which changed to reflect the fact that newer glibc is dropping the USE, meaning without the USE, it's _unavailable_, as opposed to prior to glibc-2.29 or whatever it was, where the absence of the USE meant that it's _always on_.